### DISCUSSION TITLE
Non-SysAdmin users can't see the Recalculate Rollups LWC component on page

### DISCUSSION THREAD

**Discussion by baobao917** [jamessimone/apex-rollup#502] [Q&A] [answered]
Wasn't sure if this was a bug or a config issue. Pls let me know if I need to submit as a bug issue.

I have configured my lightning record page to use the "Recalculate Rollups" LWC component.  However, only Sys Admins seem to be able to view.  Non-Sys Admins don't see this button.  I have checked the permission sets and only see what permission set related to Apex Rollup called "See Rollup App"

Side-note enhancement suggestion - Recommend standardizing the name of the Apex rollup permission sets (i.e. Apex Rollup: See Rollup App).  Would make it easier for users to clearly identify.

SysAdmin view
![image](https://github.com/jamessimone/apex-rollup/assets/104730171/eb7af4dd-70e8-432a-bc66-a7672d6f192e)

Non-SysAdmin view
![image](https://github.com/jamessimone/apex-rollup/assets/104730171/c7bc067a-f296-4fbf-8cfa-34b685a8c65a)

Permission Sets
![image](https://github.com/jamessimone/apex-rollup/assets/104730171/9898a4e1-c208-4b90-97c9-245c101b9360)

**Comment by jamessimone** [comment]
@baobao917 I'll update the naming per your recommendation. I think it probably makes sense to have two different permission sets - one for exposing the rollup app, another for exposing the button properly, but I'm curious to hear your thoughts on this. Apologies in the late response, I missed the original notification for this discussion

**Reply by baobao917** [reply]
@jamessimone I think that makes sense.  Exposing the rollup app isn't something I would want to give to all users...most likely just Sys Admins like myself.  However, the button is something that I would want to expose to most users to allow them to recalculate the rollups if for some reason they thought it was wrong or just wanted to do so for comfort.

**Reply by jamessimone** [reply, marked as answer]
@baobao917 great! that'll be included with #506
